integration/v3_grpc_test.go

```diff
@@ -567,6 +565,17 @@ func TestV3WatchMultiple(t *testing.T) {
 // TestV3WatchMultipleEventsFromCurrentRevision tests Watch APIs from current revision
 // in cases it receives multiple events.
 func TestV3WatchMultipleEventsFromCurrentRevision(t *testing.T) {
+	testV3WatchMultipleEvents(t, 0, true) // multiple events
```

Comment:
I feel it is OK to copy 10s line of code rather than trying to reuse the existing code. The reuse makes the code messy.